tests/queries/0_stateless/00405_pretty_formats.reference

```diff
@@ -211,3 +211,96 @@ Showed first 6.
      5   5       (5,'5')                 2 
 
 Showed first 6.
++-------+-------+---------+-----------------+
+| [1mhello[0m | [1mworld[0m | [1mtuple  [0m | [1msometimes_nulls[0m |
++-------+-------+---------+-----------------+
+|     0 | 0     | (0,'0') |            ᴺᵁᴸᴸ |
```

Comment:
It's funny that we use special unicode symbols for ᴺᵁᴸᴸ even for ASCII variant.
Maybe also use ASCII if user requested that?

Reply:
And also there are some symbols like &hellip;

Reply:
Added replacements for ᴺᵁᴸᴸ and …